compile failed after upgrade MacOS(intel cpu) to Sequoia(15.0)

#### System information

Geth version: v1.14.9
CL client & version: null
OS & Version: OSX Sequoia(15.0)
```txt
➜  go-ethereum git:(v1.14.9) uname -a
Darwin skydeMacBook-Pro.local 24.0.0 Darwin Kernel Version 24.0.0: Mon Aug 12 20:54:30 PDT 2024; root:xnu-11215.1.10~2/RELEASE_X86_64 x86_64
```
Commit hash : null
golang version:
```
➜  go-ethereum git:(v1.14.9) go version
go version go1.22.0 darwin/amd64
```

#### Expected behaviour
`make geth` will success

#### Actual behaviour
`make geth` failed with err:



#### Steps to reproduce the behaviour
```bash
git clone --depth 1 -b v1.14.9 https://github.com/ethereum/go-ethereum.git
cd go-ethereum
make geth
```

#### Backtrace
null

When submitting logs: please submit them as text and not screenshots.
```txt
➜  go-ethereum git:(v1.14.9) git remote -v
origin	https://github.com/ethereum/go-ethereum.git (fetch)
origin	https://github.com/ethereum/go-ethereum.git (push)
➜  go-ethereum git:(v1.14.9) git status
HEAD detached at v1.14.9
nothing to commit, working tree clean
➜  go-ethereum git:(v1.14.9) make clean
go clean -cache
rm -fr build/_workspace/pkg/ ./build/bin/*
➜  go-ethereum git:(v1.14.9) make geth
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
cmp
internal/unsafeheader
unicode/utf8
internal/race
internal/coverage/rtcov

...

github.com/cockroachdb/pebble/record
github.com/cockroachdb/pebble/internal/arenaskl
github.com/cockroachdb/pebble/internal/cache
github.com/cockroachdb/pebble/internal/manifest
github.com/cockroachdb/pebble/objstorage
github.com/cockroachdb/pebble/objstorage/objstorageprovider/objiotracing
github.com/cockroachdb/pebble/objstorage/objstorageprovider/remoteobjcat
github.com/cockroachdb/pebble/objstorage/objstorageprovider
github.com/cockroachdb/pebble/internal/keyspan
# github.com/supranational/blst/bindings/go
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:1473:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:6556:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:6674:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:6810:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:6946:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7080:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7307:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7502:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7560:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7883:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:7962:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:8090:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:8558:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:8626:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:8712:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:9159:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:9354:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:9417:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:9679:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-2bf986.s:9755:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
github.com/cockroachdb/pebble/internal/rangedel
github.com/cockroachdb/pebble/internal/private
github.com/cockroachdb/pebble/internal/rangekey
github.com/ethereum/go-ethereum/internal/jsre
github.com/ethereum/go-ethereum/crypto
github.com/ethereum/go-ethereum/crypto/ecies
github.com/ethereum/go-ethereum/trie/trienode
github.com/ethereum/go-ethereum/p2p/enode
github.com/ethereum/go-ethereum/accounts/abi
github.com/ethereum/go-ethereum/p2p/rlpx
github.com/ethereum/go-ethereum/p2p/discover/v4wire
github.com/ethereum/go-ethereum/p2p/dnsdisc
github.com/ethereum/go-ethereum/p2p/discover/v5wire
github.com/ethereum/go-ethereum/p2p/discover
github.com/ethereum/go-ethereum/p2p
github.com/influxdata/influxdb-client-go/v2/api/write
github.com/influxdata/influxdb-client-go/v2/internal/write
github.com/influxdata/influxdb-client-go/v2/api
github.com/influxdata/influxdb-client-go/v2
github.com/ethereum/go-ethereum/metrics/influxdb
github.com/cockroachdb/pebble/sstable
github.com/cockroachdb/pebble/rangekey
github.com/cockroachdb/pebble
github.com/ethereum/go-ethereum/ethdb/pebble
util.go:47: exit status 1
exit status 1
make: *** [geth] Error 1
```

after upgrade the newest commandline tools, compile failed:
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) uname -a
Darwin skydeMacBook-Pro.local 24.0.0 Darwin Kernel Version 24.0.0: Mon Aug 12 20:54:30 PDT 2024; root:xnu-11215.1.10~2/RELEASE_X86_64 x86_64
➜  go-ethereum git:(v1.14.9) go version
go version go1.22.7 darwin/amd64
➜  go-ethereum git:(v1.14.9) pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
package-id: com.apple.pkg.CLTools_Executables
version: 16.0.0.0.1.1724870825
volume: /
location: /
install-time: 1727243166
➜  go-ethereum git:(v1.14.9) softwareupdate -l
Software Update Tool

Finding available software
No new software available.
➜  go-ethereum git:(v1.14.9) git remote -v
origin	https://github.com/ethereum/go-ethereum.git (fetch)
origin	https://github.com/ethereum/go-ethereum.git (push)
➜  go-ethereum git:(v1.14.9) git status
HEAD detached at v1.14.9
nothing to commit, working tree clean
➜  go-ethereum git:(v1.14.9) make clean;make geth
go clean -cache
rm -fr build/_workspace/pkg/ ./build/bin/*
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
internal/unsafeheader

......

github.com/cockroachdb/pebble/vfs
# github.com/supranational/blst/bindings/go
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:1473:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:6556:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:6674:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:6810:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:6946:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7080:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7307:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7502:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7560:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7883:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:7962:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:8090:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:8558:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:8626:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:8712:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:9159:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:9354:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:9417:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:9679:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-eecf3f.s:9755:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
github.com/cockroachdb/pebble/vfs/atomicfs

...

github.com/ethereum/go-ethereum/ethdb/pebble
util.go:47: exit status 1
exit status 1
make: *** [geth] Error 1


delete CommandLineTools and reinstall it with cmd `xcode-select --install`:
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) sudo rm -rf /Library/Developer/CommandLineTools
Password:
➜  go-ethereum xcode-select --install
xcode-select: note: install requested for command line developer tools
➜  go-ethereum pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
package-id: com.apple.pkg.CLTools_Executables
version: 15.3.0.0.1.1708646388
volume: /
location: /
install-time: 1727245698
➜  go-ethereum git:(v1.14.9) softwareupdate -l
Software Update Tool

Finding available software
Software Update found the following new or updated software:
* Label: Command Line Tools for Xcode-16.0
	Title: Command Line Tools for Xcode, Version: 16.0, Size: 751012KiB, Recommended: YES, 

`make geth` again it will succeed
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) make clean;make geth
go clean -cache
rm -fr build/_workspace/pkg/ ./build/bin/*
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
internal/itoa
internal/goarch

...

github.com/ethereum/go-ethereum/beacon/blsync
github.com/ethereum/go-ethereum/cmd/geth
Done building.
Run "./build/bin/geth" to launch geth.
➜  go-ethereum git:(v1.14.9) echo $?
0


then upgrade `CommandLineTools` complete, `make geth` will fail
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
package-id: com.apple.pkg.CLTools_Executables
version: 16.0.0.0.1.1724870825
volume: /
location: /
install-time: 1727246236
➜  go-ethereum git:(v1.14.9) softwareupdate -l
Software Update Tool

Finding available software
No new software available.

make geth again:
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) make clean;make geth
go clean -cache
rm -fr build/_workspace/pkg/ ./build/bin/*
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
internal/race
internal/goos

......

# github.com/supranational/blst/bindings/go
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:1473:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:6556:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:6674:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:6810:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:6946:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7080:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7307:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7502:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7560:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7883:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:7962:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:8090:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:8558:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:8626:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:8712:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:9159:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:9354:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:9417:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:9679:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-84cc3f.s:9755:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
github.com/cockroachdb/pebble/vfs/atomicfs

......

github.com/ethereum/go-ethereum/ethdb/pebble
util.go:47: exit status 1
exit status 1
make: *** [geth] Error 1
➜  go-ethereum git:(v1.14.9) make geth
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
github.com/supranational/blst/bindings/go
# github.com/supranational/blst/bindings/go
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:1473:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:6556:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:6674:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:6810:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:6946:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7080:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7307:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7502:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7560:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7883:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:7962:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:8090:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:8558:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:8626:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:8712:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:9159:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:9354:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:9417:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:9679:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
/var/folders/rd/m8gg8hzs66b1ccvvr730799h0000gn/T/cgo_assembly-ffae11.s:9755:1: error: invalid CFI advance_loc expression
.cfi_adjust_cfa_offset 8
^
util.go:47: exit status 1
exit status 1
make: *** [geth] Error 1


update blst from v0.3.11 to v0.3.13:
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) vi go.mod
➜  go-ethereum git:(v1.14.9) ✗ go mod tidy    
➜  go-ethereum git:(v1.14.9) ✗ git diff > diff
➜  go-ethereum git:(v1.14.9) ✗ cat diff
diff --git a/go.mod b/go.mod
index a8f5d11a8..e9692cf8b 100644
--- a/go.mod
+++ b/go.mod
@@ -61,7 +61,7 @@ require (
 	github.com/shirou/gopsutil v3.21.4-0.20210419000835-c7a38de76ee5+incompatible
 	github.com/status-im/keycard-go v0.2.0
 	github.com/stretchr/testify v1.9.0
-	github.com/supranational/blst v0.3.11
+	github.com/supranational/blst v0.3.13
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/tyler-smith/go-bip39 v1.1.0
 	github.com/urfave/cli/v2 v2.25.7
diff --git a/go.sum b/go.sum
index ab9bff996..21a5e5bcd 100644
--- a/go.sum
+++ b/go.sum
@@ -495,8 +495,8 @@ github.com/stretchr/testify v1.5.1/go.mod h1:5W2xD1RspED5o8YsWQXVCued0rvSQ+mT+I5
 github.com/stretchr/testify v1.7.0/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/stretchr/testify v1.9.0 h1:HtqpIVDClZ4nwg75+f6Lvsy/wHu+3BoSGCbBAcpTsTg=
 github.com/stretchr/testify v1.9.0/go.mod h1:r2ic/lqez/lEtzL7wO/rwa5dbSLXVDPFyf8C91i36aY=
-github.com/supranational/blst v0.3.11 h1:LyU6FolezeWAhvQk0k6O/d49jqgO52MSDDfYgbeoEm4=
-github.com/supranational/blst v0.3.11/go.mod h1:jZJtfjgudtNl4en1tzwPIV3KjUnQUvG3/j+w+fVonLw=
+github.com/supranational/blst v0.3.13 h1:AYeSxdOMacwu7FBmpfloBz5pbFXDmJL33RuwnKtmTjk=
+github.com/supranational/blst v0.3.13/go.mod h1:jZJtfjgudtNl4en1tzwPIV3KjUnQUvG3/j+w+fVonLw=
 github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7 h1:epCh84lMvA70Z7CTTCmYQn2CKbY8j86K7/FAIr141uY=
 github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7/go.mod h1:q4W45IWZaF22tdD+VEXcAWRA037jwmWEB5VWYORlTpc=
 github.com/tklauser/go-sysconf v0.3.12 h1:0QaGUFOdQaIVdPgfITYzaTegZvdCjmYO52cSFAEVmqU=


`make geth` again, it will success:
----------------------------------------------------------------------------
➜  go-ethereum git:(v1.14.9) ✗ make clean;make geth
go clean -cache
rm -fr build/_workspace/pkg/ ./build/bin/*
go run build/ci.go install ./cmd/geth
>>> /usr/local/go/bin/go build -ldflags "--buildid=none -X github.com/ethereum/go-ethereum/internal/version.gitCommit=c350d3acd57a543b410e9e16598488183bcd5030 -X github.com/ethereum/go-ethereum/internal/version.gitDate=20240918 -s" -tags urfave_cli_no_docs,ckzg -trimpath -v -o /Users/sky/go-ethereum/build/bin/geth ./cmd/geth
encoding
internal/goarch
internal/unsafeheader
internal/goos

......

github.com/ethereum/go-ethereum/cmd/geth
Done building.
Run "./build/bin/geth" to launch geth.
➜  go-ethereum git:(v1.14.9) ✗ echo $?
0
➜  go-ethereum git:(v1.14.9) ✗

--- a/go.mod
+++ b/go.mod
@@ -61,7 +61,7 @@ require (
 	github.com/shirou/gopsutil v3.21.4-0.20210419000835-c7a38de76ee5+incompatible
 	github.com/status-im/keycard-go v0.2.0
 	github.com/stretchr/testify v1.9.0
-	github.com/supranational/blst v0.3.11
+	github.com/supranational/blst v0.3.13
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/tyler-smith/go-bip39 v1.1.0
 	github.com/urfave/cli/v2 v2.25.7

